docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1199 @@
+---
+title: Migrate to the V2 primitives
+description: Migrate from using the original primitives (Estimator and Sampler) to using the updated V2 primitives (EstimatorV2 and SamplerV2)
+
+---
+
+<span id="v2-primitives"></span>
+# Migrate to the Qiskit Runtime V2 primitives
+
+This guide describes what changed in the Qiskit Runtime V2 primitives and why, describes each new primitive in detail, and gives examples to help you migrate code from using the legacy primitives to the V2 primitives.  The examples in the guide all use the Qiskit Runtime primitives, but in general, the same changes apply to the other primitive implementations.  The functions unique to Qiskit Runtime such as error mitigation remain unique to Qiskit Runtime. For information about the changes to the Qiskit reference primitives (now called *statevector* primitives), see (the qiskit.primitives section)[qiskit-1.0-features#qiskit.primitives] in the Qiskit 1.0 feature changes page. 
+
+## Overview
+
+Version 2 of the primitives is introduced with a new base class for both Sampler and Estimator ([BaseSamplerV2](/api/qiskit/qiskit.primitives.BaseSamplerV2) and [BaseEstimatorV2](/api/qiskit/qiskit.primitives.BaseEstimatorV2)), along with new types for their inputs and outputs. The emphasis of this new version is on performing vectorized calls to the primitive `.run()` methods so that you can efficiently specify sweeps over parameter value sets and observables. See [StatevectorSampler](/api/qiskit/qiskit.primitives.StatevectorSampler) and [StatevectorEstimator](/api/qiskit/qiskit.primitives.StatevectorEstimator) for V2 primitive reference implementations.
+
+Based on user feedback, this version introduces the following major new functions:
+
+* A new interface that lets you specify a single circuit and multiple observables (if using Estimator) and parameter value sets for that circuit. Previously, you had to specify the  same circuit multiple times to match the size of the data to be combined.
+* The ability to turn on or off individual error mitigation / suppression methods.
+* SamplerV2 is simplified to focus on its core task of sampling the output register from execution of quantum circuits. This has resulted the the following specific changes: 
+  - It returns the samples, whose type is defined by the program, without weights and therefore no longer supports resilience levels. 
+  - The output data is separated by the output register names and has convenience methods like `get_counts()` to help with migration. This also enables future support for circuits with classical control flow.
+  - Returns the outputs (bitstrings) from every shot, rather than providing a counts-like return (quasi-probabilities). The result class, however, has methods to return weighted samples, such as counts.
+  - Has a new `shots` argument in the `run()` method that enables you to specify different shots for each input block. 
```

Comment:
The key message for SamplerV2 output is really that
1. It does just sampling with no post processing
2. It obeys the output type and names defined in the program, which makes it compatible with OpenQASM3 specification. 

So maybe something like

SamplerV2 is simplified to focus on its core task of sampling the output register from execution of quantum circuits. It returns the samples, whose type is defined by the program, without weights. The output data is also separated by the output register names defined by the program. This change enables future support for circuits with classical control flow.